sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableClient.java

```diff
@@ -78,26 +78,199 @@
 /**
  * Provides a synchronous service client for accessing a table in the Azure Tables service.
  *
+ * <h2>Overview</h2>
+ *
  * <p>The client encapsulates the URL for the table within the Tables service endpoint, the name of the table, and the
- * credentials for accessing the storage or CosmosDB table API account. It provides methods to create and delete the
+ * credentials for accessing the storage or CosmosDB table API account. It provides synchronous methods to create and delete the
  * table itself, as well as methods to create, upsert, update, delete, list, and get entities within the table. These
  * methods invoke REST API operations to make the requests and obtain the results that are returned.</p>
  *
- * <p>Instances of this client are obtained by calling the {@link TableClientBuilder#buildClient()} method on a
- * {@link TableClientBuilder} object.</p>
+ * <h2>Getting Started</h2>
+ *
+ * <p>Authenticating and building instances of this client are handled by {@link TableClientBuilder}.
+ * This sample shows how to authenticate and build a TableClient instance using the {@link TableClientBuilder} and
+ * a connection string.</p>
  *
- * <p><strong>Samples to construct a sync client</strong></p>
- * <!-- src_embed com.azure.data.tables.tableClient.instantiation -->
+ * <!-- src_embed com.azure.data.tables.tableClient.connectionstring.instantiation -->
  * <pre>
  * TableClient tableClient = new TableClientBuilder&#40;&#41;
- *     .endpoint&#40;&quot;https:&#47;&#47;myaccount.core.windows.net&#47;&quot;&#41;
- *     .credential&#40;new AzureNamedKeyCredential&#40;&quot;name&quot;, &quot;key&quot;&#41;&#41;
+ *     .connectionString&#40;&quot;connectionstring&quot;&#41;
  *     .tableName&#40;&quot;myTable&quot;&#41;
  *     .buildClient&#40;&#41;;
  * </pre>
- * <!-- end com.azure.data.tables.tableClient.instantiation -->
+ * <!-- end com.azure.data.tables.tableClient.connectionstring.instantiation -->
+ *
+ * <p>For more information on building and authenticating, see the {@link TableClientBuilder} documentation.</p>
+ *
+ * <p>The following code samples provide examples of common operations preformed with this client.</p>
+ *
```

Comment:
Add here and for other samples,
 * <p><strong>Note:</strong> for asynchronous sample, refer to {@link async client}</p>